source/less/blocks/control.less

```
@@ -0,0 +1,24 @@
+.control {
+  position: relative;
+}
+
+.control-circle {
```

```suggestion
.control__circle {
```
Логичнее, если это будет класс элемента.